src/network/bmproto.py

```diff
@@ -441,9 +441,9 @@ def bm_command_addr(self):
             if stream not in state.streamsInWhichIAmParticipating:
                 continue
             if (
-                decodedIP and time.time() - seenTime > 0 and
-                seenTime > time.time() - ADDRESS_ALIVE and
-                port > 0
+                decodedIP and time.time()
+                - seenTime > 0 and seenTime > time.time()
+                - ADDRESS_ALIVE and port > 0
```

Comment:
I would rearrange it so that the lines begin with and:

```
decodedIP
and time.time() - seenTime > 0
and seenTime > time.time() - ADDRESS_ALIVE
and port > 0
```